recipes/chainer/meta.yaml

```diff
@@ -0,0 +1,105 @@
+{% set name = "chainer" %}
+{% set version = "7.1.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: afe4531a5b1b95449a49f06c6d12c881a39a603067bc067e0af89849692f4674
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: python
+
+requirements:
+  host:
+    - pip
+    - python >=3.5
+
+  run:
+    - filelock
+    - numpy >=1.9.0
+    - protobuf >=3.0
+    - python >=3.5
+    - six >=1.9.0
+    - typing_extensions
+
+test:
+  imports:
+    - chainer
+    - chainer.backends
+    - chainer.dataset
+    - chainer.dataset.tabular
+    - chainer.datasets
+    - chainer.distributions
+    - chainer.exporters
+    - chainer.function_hooks
+    - chainer.functions
+    - chainer.functions.activation
+    - chainer.functions.array
+    - chainer.functions.connection
+    - chainer.functions.evaluation
+    - chainer.functions.loss
+    - chainer.functions.math
+    - chainer.functions.noise
+    - chainer.functions.normalization
+    - chainer.functions.pooling
+    - chainer.functions.rnn
+    - chainer.functions.theano
+    - chainer.functions.util
+    - chainer.graph_optimizations
+    - chainer.initializers
+    - chainer.iterators
+    - chainer.link_hooks
+    - chainer.links
+    - chainer.links.activation
+    - chainer.links.caffe
+    - chainer.links.caffe.protobuf3
+    - chainer.links.connection
+    - chainer.links.loss
+    - chainer.links.model
+    - chainer.links.model.vision
+    - chainer.links.normalization
+    - chainer.links.rnn
+    - chainer.links.theano
+    - chainer.optimizer_hooks
+    - chainer.optimizers
+    - chainer.serializers
+    - chainer.testing
+    - chainer.training
+    - chainer.training.extensions
+    - chainer.training.triggers
+    - chainer.training.updaters
+    - chainer.utils
+    - chainermn
+    - chainermn.communicators
+    - chainermn.datasets
+    - chainermn.extensions
+    - chainermn.functions
+    - chainermn.iterators
+    - chainermn.links
+    - chainermn.testing
+    - chainerx
+
+  requires:
+    - attrs <19.2.0
+    - mock
+    - pytest <4.2.0
```

Comment:
These requirements look like test requirements but shouldn't be used for the import tests, please remove.

Reply:
@xhochy thanks, test reqs have been removed.